src/dex-ui/routes/routes.ts

```diff
@@ -23,7 +23,17 @@ export const Paths = {
     absolute: "/daos",
     default: "daos",
     Create: "create",
-    DAODetails: "daos",
+    DAODetails: "details",
```

Comment:
@damiano-melcarne-lab49  do we require keeping routes path name this(object's) way, I believe this will allocate extra memory to store the paths in objects.? we could directly give the paths by strings in `<Route>` like rest other UI.

Reply:
Using an object is useful since we can maintain and update paths in a single location. Using strings opens up room for error since it's easy to mistype a string path in a function such as `navigate` from `react-router`. `navigate` does not have type checking against string path inputs.